bom/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
         parent.childProjects.sort { "$it.value" }
                 .collect { it.value }
                 .findAll { it.name != project.name }
+                .findAll { !it.name.endsWith('-metrics') }
```

Comment:
Does `it.version.endsWith('-alpha')` work?

Reply:
maybe `.contains`, since SNAPSHOT always ends up last.

Reply:
It doesn't work.  It seems to be the case that the dependency resolution happens early enough that the version has not been customized in the submodule yet...or at least not consistently/reliably.

Reply:
Too bad - thanks for checking!